src/index.html

```diff
@@ -8,6 +8,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+    <div class="stars stars--1">
+      <div class="stars__star star--active"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+    <div class="stars stars--2">
+      <div class="stars__star star--active"></div>
+      <div class="stars__star star--active"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+    <div class="stars stars--3">
+      <div class="stars__star star--active"></div>
+      <div class="stars__star star--active"></div>
+      <div class="stars__star star--active"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+    <div class="stars stars--4">
+      <div class="stars__star star--active"></div>
+      <div class="stars__star star--active"></div>
+      <div class="stars__star star--active"></div>
+      <div class="stars__star star--active"></div>
+      <div class="stars__star"></div>
+    </div>
+    <div class="stars stars--5">
+      <div class="stars__star star--active"></div>
+      <div class="stars__star star--active"></div>
+      <div class="stars__star star--active"></div>
+      <div class="stars__star star--active"></div>
+      <div class="stars__star star--active"></div>
+    </div>
```

Comment:
Using `stars--active` is not a correct solution. You should rely only on `stars--5` in this case. For that you can use `:nth-child` selector.
```suggestion
    <div class="stars stars--5">
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
    </div>
```

Reply:
Done! Thank you for your feedback